r/tests/testthat/test-Array.R

```diff
@@ -18,16 +18,16 @@
 context("Array")
 
 expect_array_roundtrip <- function(x, type) {
-  a <- Array$create(x)
+  a <- Array$create(x, type = type)
   expect_type_equal(a$type, type)
   expect_identical(length(a), length(x))
-  if (!inherits(type, "ListType")) {
+  if (!inherits(type, "ListType") && !inherits(type, "LargeListType")) {
```

Comment:
Can/should LargeListType inherit from ListType?

```suggestion
  if (!inherits(type, c("ListType", "LargeListType"))) {
```

Reply:
I don't think there is inheritance down the C++ code